### PR TITLE
Add kicad-doc zh

### PR DESCRIPTION
**Pull request by taotieren**

### CODE REVIEW

**Comment by taotieren** [comment]
@nickoe I added the Chinese configuration file for kicad-doc and also added an introduction to Chinese installation.

File: nsis/install.nsi [hunk 1]
```diff
@@ -423,6 +425,11 @@ SectionGroup $(TITLE_SEC_DOCS) SEC06
     SetOutPath "$INSTDIR\share\doc\kicad\help\pl"
     File /nonfatal /r "..\share\doc\kicad\help\pl\*"
   SectionEnd
+    Section $(LANGUAGE_NAME_ZH) SEC06_ZH
```
Comment (nickoe) [inline nsis/install.nsi]: Please fix the indentation, that is remove the two spaces

Comment (taotieren) [reply nsis/install.nsi]: OK,I've fixed it.

**Comment by nickoe** [comment]
Please verify on next nightly.

**Comment by taotieren** [comment]
@nickoe Ok, I will verify.thanks

**Pull request merged**